pocs/camera/camera.py

```diff
@@ -319,8 +318,6 @@ def autofocus(self,
                 keyword arguments for the merit function.
             mask_dilations (int, optional): Number of iterations of dilation to perform on the
                 saturated pixel mask (determine size of masked regions), default 10
-            spline_smoothing (float, optional): smoothing parameter for the spline fitting to
```

Comment:
FWIW, these several spline changes seem unrelated to adding the distributed camera support.

Reply:
Yes, leftover arguments and docstring lines that should have been removed back when the spline fitting was removed. Don't really belong in this PR but couldn't resist pruning them.